src/validation.h

```diff
@@ -206,12 +206,26 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * (Try to) add a transaction to the memory pool.
- * @param[in]  bypass_limits   When true, don't enforce mempool fee limits.
- * @param[in]  test_accept     When true, run validation checks but don't submit to mempool.
+ * Try to add a transaction to the mempool with a specified acceptance time. This is an internal
+ * function and is exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
+ *
+ * @param[in]  chainparams        Chain paramenters.
+ * @param[in]  pool               Reference to the node's mempool.
+ * @param[in]  active_chainstate  Reference to the active chainstate.
+ * @param[in]  tx                 The transaction to submit for mempool acceptance.
+ * @param[in]  accept_time        The timestamp for adding the transaction to the mempool. Usually
+ *                                the current system time, but may be different (eg when loading
+ *                                mempool txs from disk on startup).
+ *                                It is also used to determine when the entry expires.
+ * @param[in]  bypass_limits      When true, don't enforce mempool fee and capacity limits.
+ * @param[in]  test_accept        When true, run validation checks but don't submit to mempool.
+ *
+ * @returns a MempoolAcceptResult indicating whether the transaction was accepted/rejected with reason.
  */
-MempoolAcceptResult AcceptToMemoryPool(CChainState& active_chainstate, CTxMemPool& pool, const CTransactionRef& tx,
-                                       bool bypass_limits, bool test_accept=false) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+MempoolAcceptResult AcceptToMemoryPool(const CChainParams& chainparams, CTxMemPool& pool,
+                                                        CChainState& active_chainstate, const CTransactionRef &tx,
+                                                        int64_t accept_time, bool bypass_limits, bool test_accept)
```

Comment:
Please align arguments on new lines after the opening parens:

```suggestion
MempoolAcceptResult AcceptToMemoryPool(const CChainParams& chainparams, CTxMemPool& pool,
                                       CChainState& active_chainstate, const CTransactionRef &tx,
                                       int64_t accept_time, bool bypass_limits, bool test_accept)
```

(We use clang-style `AlignAfterOpenBracket: true`)

Reply:
Done in 9a0b0fc